### ISSUE TITLE
Feature: Copy vault UUID and item UUID via desktop apps

### ISSUE THREAD

**Issue by SimonBarendse** [1Password/onepassword-sdk-python#101]
### Use Case

Trying to fetch a known item using the SDK:

```python
item = await client.items.get(vault_id, item_id)
```

When you know the vault name and item name, but don't know the UUIDs, it's currently cumbersome to look these up. Would be a great UX improvement if we could fetch UUIDs from inside the desktop apps.

### Requirements and desired behavior

Would be great if there's a button in the app to copy the vault UUID and item UUID. For example here in the actions menu on an item:
![image](https://github.com/user-attachments/assets/2f27c53d-24c5-4753-9860-569b6219eaf1)

### Additional information

Related issues (use cases): #99 and #100.

**Comment by simkimsia** [comment]
Wouldn't this actually be a ticket for the desktop app team?

**Comment by SimonBarendse** [comment]
Yes, you're right that we won't tackle this in this repo. We have generalists teams though that cover domains (in our case: Programmatic Access) and we're not silo'ed to specific products or codebases. So our team here can tackle this. I've filed the ticket in this repository for public visibility, since we do not have a public issue tracker for our closed source desktop app repo.

**Comment by simkimsia** [comment]
> we're not silo'ed to specific products or codebases. So our team here can tackle this. 

This is an awesome setup

**Comment by ohinibla** [comment]
Isn't there already a button for copying item's UUID? (I'm using the Linux app on Fedora)

![2024-08-15-223539_hyprshot](https://github.com/user-attachments/assets/439798dd-5248-47e5-8496-9e19d438e462)

getting a similar button for vault's also would be nice.
also a temporary workaround for getting the vault's UUID and item's UUID to be able to use it in the SDK would be that the ids are encoded in the private link as v={vault_id} and i={item_id}
i.e:
start.1password.com/open/i?a=XXXXXXXXXXXXXXXX&**v={vault_id}**&**i={item_id}**&h=my.1password.eu

**Comment by hculea** [comment]
This has now been added in the desktop app, and is available by enabling `Advanced > Show Debugging Tools` or `Developer > Show 1Password Developer Experience`.

<img width="298" alt="Screenshot 2025-01-10 at 09 56 21" src="https://github.com/user-attachments/assets/bbd619f7-7064-4b54-a7da-bad9bdd35ed5" />

You can find this feature right-clicking on the vault in the left-hand side overview. 

I will be closing this ticket for now, but please do reopen if needed!

cc @simkimsia @ohinibla
